webshopping/lolshopping/templates/design/main.html

```diff
@@ -132,7 +133,37 @@
               </div>
 
             </div>
+            <script>
 
+            $("#choices").keyup(function() {
+
+              // Retrieve the input field text and reset the count to zero
+              var filter = $(this).val(), count = 0;
+              var category = document.getElementById("categories");
+              var choice = document.getElementById("choices");
+
+
+              // Loop through the comment list
+              if (category.value == "name" || category.value == "role" ){
+
+              $('#results div').each(function() {
+
+
+                // If the list item does not contain the text phrase fade it out
+                if ($(this).text().search(new RegExp(filter, "i")) < 0) {
+                  $(this).hide();  // MY CHANGE
```

Comment:
Remove "// MY CHANGE" and the line should be tabbed and not spaced.

Reply:
sure do